### PR TITLE
Add support for Fallback Locales

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: lib/contentful/management/entry.rb [hunk 1]
```diff
@@ -108,12 +108,13 @@ def locale
       # Returns a hash of existing fields.
       #
       # @private
-      def fields_for_query
+      def fields_for_query(remove_undefined = true)
```
Comment (jsdalton) [inline lib/contentful/management/entry.rb]: Does it make sense to add a comment, either as a docstring or even down below? Specifically it'd be useful to be more explicit about what `remove_undefined` means, or what purpose it serves.

File: spec/lib/contentful/management/entry_spec.rb [hunk 2]
```diff
@@ -810,6 +810,26 @@ class Contentful::BaseEntry < Contentful::Entry
         end
       end
 
+      describe 'fallback locales' do
+        let(:space) { client.spaces.find('wqjq16zu9s8b') }
+        it 'if a property is nil, its removed from the request to undefine it in the API' do
```
Comment (jsdalton) [inline spec/lib/contentful/management/entry_spec.rb]: its => it's

**Comment by dlitvakb** [comment]
@jsdalton done 👍

**Comment by jsdalton** [comment]
👍  LGTM

**Pull request merged**